docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -364,18 +364,18 @@ Also, see [Types for compiler options](#types-for-compiler-options).
 
 ### Attributes specific to JavaScript
 
-| Name | Description | Possible values |Default value |
-|---|---|---|---|
-| `friendModulesDisabled` | Disable internal declaration export| | false |
-| `main` | Define whether the `main` function should be called upon execution | "call", "noCall". Also, see [Types for compiler options](#types-for-compiler-options) | "call" |
-| `metaInfo` | Generate .meta.js and .kjsm files with metadata. Use to create a library | | true |
-| `moduleKind` | The kind of JS module generated by the compiler | "umd", "commonjs", "amd", "plain", "es". Also, see [Types for compiler options](#types-for-compiler-options) | "umd" |
-| `sourceMap` | Generate source map | | true |
-| `sourceMapEmbedSources` | Embed source files into the source map | "never", "always", "inlining". Also, see [Types for compiler options](#types-for-compiler-options) |  |
-| `sourceMapNamesPolicy` | Add variable and function names that you declared in Kotlin code into the source map. For more information on the behavior, see our [compiler reference](compiler-reference.md#source-map-names-policy-simple-names-fully-qualified-names-no). | "simple-names", "fully-qualified-names", "no". Also, see [Types for compiler options](#types-for-compiler-options)                                                              | "simple-names" |
-| `sourceMapPrefix` | Add the specified prefix to paths in the source map |  |  |
-| `target` | Generate JS files for specific ECMA version  | "v5"  | "v5" |
-| `typedArrays` | Translate primitive arrays to JS typed arrays | | true |
+| Name | Description                                                                                                                                                                                                                              | Possible values | Default value               |
+|---|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---|-----------------------------|
+| `friendModulesDisabled` | Disable internal declaration export                                                                                                                                                                                                      | | false                       |
+| `main` | Define whether the `main` function should be called upon execution                                                                                                                                                                       | "call", "noCall". Also, see [Types for compiler options](#types-for-compiler-options) | JsMainFunctionExecutionMode.CALL |
```

Comment:
As it is about Gradle DSL - I would propose not to use strings ("call", "noCall") but rather actual possible types: `JsMainFunctionExecutionMode.CALL` and `JsMainFunctionExecutionMode.NO_CALL`.

Same approach should applied in changes below.